src/D2L.CodeStyle.Analyzers/ApiUsage/Events/EventHandlersDisallowedListAnalyzer.cs

```diff
@@ -41,11 +41,8 @@ internal sealed class EventHandlersDisallowedListAnalyzer : DiagnosticAnalyzer {
 				INamedTypeSymbol type
 			) {
 
-			SimpleBaseTypeSyntax baseTypeSyntax = (SimpleBaseTypeSyntax)context.Node;

```

Comment:
Merge conflict with my change to `RegisterSymbolAction`.  Just took what's in master